README.md

```diff
@@ -65,9 +65,37 @@ job_details = client.get_job_details(job.id)
 `job_details` will contain all information normally found in a successful response from
 our [Get Job](https://www.rev.ai/docs#operation/GetJobById) endpoint
 
+### Checking multiple files
+
+You can retrieve a list of transcription jobs with optional parameters
+
+```python
+jobs = client.get_list_of_jobs()
+
+# limit amount of retrieved jobs
+jobs = client.get_list_of_jobs(limits=3)
+
+# get jobs starting after a certain job id
+jobs = client.get_list_of_jobs(starting_after='3')
```

Comment:
I think '3' is a bad example here since it implies this argument is a number of jobs to start after as opposed to what it actually is: a jobid to start after. I would either add a comment or change the value to something like `another_job.id`

Reply:
Or you can even use a real jobId as an example.